### ISSUE TITLE
Testimonial section

### ISSUE THREAD

**Issue by apoorva-mk** [acm-w-nitk/acm-w-nitk.github.io#76]
## Problem
Add a testimonial section where we can feature testimonials from our event attendees.

## Solution

It should be a section under our main page and should have some kind of animations.

**Comment by Muskan284** [comment]
can i work on it?

**Comment by Muskan284** [comment]
can u tell me about content that will be  present in the testimonial

**Comment by adarshbalu** [comment]
Can I help you out with this?

**Comment by apoorva-mk** [comment]
@Muskan284 yes please work on it, make sure to go through our contributing guidelines before sending a PR.

**Comment by apoorva-mk** [comment]
> can u tell me about content that will be present in the testimonial

@Muskan284, for now, please add some dummy text.

**Comment by mridul-arora** [comment]
I have built the Testimonials Section. Do tell me if you like it :)

**Comment by mridul-arora** [comment]
And sorry, if I overstepped. It's my first PR, I didn't know that you have to assign me first

**Comment by apoorva-mk** [comment]
Update: We would like the testimonial section to look like [this](https://www.b3multimedia.ie/15-testimonial-page-examples-youll-want-to-copy-in-2018/), refer the first example. If someone can do it I will not raise it as a separate issue.

**Comment by mridul-arora** [comment]
> Update: We would like the testimonial section to look like [this](https://www.b3multimedia.ie/15-testimonial-page-examples-youll-want-to-copy-in-2018/), refer the first example. If someone can do it I will not raise it as a separate issue. Sorry for not informing earlier.

You are asking for the orange background one, OR the testimonial example named Desk?

**Comment by apoorva-mk** [comment]
> > Update: We would like the testimonial section to look like [this](https://www.b3multimedia.ie/15-testimonial-page-examples-youll-want-to-copy-in-2018/), refer the first example. If someone can do it I will not raise it as a separate issue. Sorry for not informing earlier.
> 
> You are asking for the orange background one, OR the testimonial example named Desk?

@mridul-arora hi, it's the one with the orange background.

**Comment by mridul-arora** [comment]
> > > Update: We would like the testimonial section to look like [this](https://www.b3multimedia.ie/15-testimonial-page-examples-youll-want-to-copy-in-2018/), refer the first example. If someone can do it I will not raise it as a separate issue. Sorry for not informing you earlier.
> > 
> > 
> > You are asking for the orange background one, OR the testimonial example named Desk?
> 
> @mridul-arora hi, it's the one with the orange background.

It won't look good with the current theme you have set for your website. If you want something like that or something with a much better design for every section you have raised an issue for, You should try working on a readymade html5/bootstrap template, instead of from scratch and adding one component at a time. (Just a suggestion)
